### DISCUSSION TITLE
Sugestão

### DISCUSSION THREAD

**Discussion by sygecom** [marcosgambeta/HWGUI#1] [Ideas]
Mestre,
Poderia deixar essa versão da Hwgui compatível com Mingw64 ?

**Comment by marcosgambeta** [comment]
O código atual está compilando  com Harbour e MinGW64 sem erros e sem avisos (warnings). Usei a versão 8.1 nos testes. Caso se depare com algo nos seus testes, basta informar em 'Issues', que revisarei o mais breve possível. Aqui estou testando simultaneamente em 32-bit e 64-bit, para garantir que as duas opções estejam funcionando corretamente.

**Reply by sygecom** [reply]
Deu certo aqui sim mestre.
